imgparse/imgparse.py

```diff
@@ -652,8 +652,8 @@ def get_wavelength_data(image_path, exif_data=None, xmp_data=None):
             central_wavelength = parse_seq(xmp_data[xmp_tags.WAVELENGTH_CENTRAL], int)
             wavelength_fwhm = parse_seq(xmp_data[xmp_tags.WAVELENGTH_FWHM], int)
```

Comment:
We should change these to floats too to keep it consistent

Reply:
@ajmcgraw What are your thoughts? are you okay with everything looking like this? 
![image](https://github.com/SenteraLLC/py-image-metadata-parser/assets/48452875/86ea1933-daa6-4f6a-84df-d7d118f3e683)

Reply:
Got it...  I can open that up!

Reply:
Any particular reason not to want the .0?

Reply:
Idk I guess it's not representative of what is stored in the exif data then?  Maybe it's not a huge issue...

Reply:
It's not a huge issue at all, it can go either way